RedisRepo/RedisContext.cs

```diff
@@ -91,6 +92,11 @@ private void InitConnections(RedisConnectionOptions options)
             {
                 this.connections[i] = ConnectionMultiplexer.Connect(options.RedisConfigurationOptions);
                 this.connections[i].PreserveAsyncOrder = false;
+
+                if (!this.connections[i].IsConnected)
+                {
+                    throw new ConnectionFailedException("Failed connecting to Resdis server - please check connection details");

```

Comment:
fix Resdis
Move the message to be part of the ConnectionFailedException constructor